### ISSUE TITLE
StridedSequenceDataset target indexing

### ISSUE THREAD

**Issue by maycherryblossom** [Sachini/ronin#55]
Hi, I’ve been working with your RoNIN model (ResNet) and had a question regarding data indexing for feature and target during training.

In the paper, it mentions fitting the velocity at the i-th frame with features from frames i-200 to i. 
However, in the provided code, particularly in `StridedSequenceDataset`, the dataset indexing seems different:

`feat = self.features[seq_id][frame_id:frame_id + self.window_size]`
`targ = self.targets[seq_id][frame_id]`

This appears to fit the velocity at the i-th frame with the features from frames i to i+200, rather than i-200 to i as described in the paper. Could you clarify how the data is being indexed and why there seems to be this difference?

Thanks!

**Comment by csr818** [comment]
Hi, I found a similar problem. How did you solve it? Have you tried modifying the index? 
Thanks!
@maycherryblossom

**Comment by maycherryblossom** [comment]
@csr818 
Hello! Yes, I modified the index for training, and it worked well for me. I shifted the start and end ranges of the frame IDs by the window size, and indexed the features using [frame_id - self.window_size:frame_id].

**Comment by csr818** [comment]
Thanks for reply!



Before modify the index, I have finished training with the hyperparameters mentioned in the paper.
I found that the train loss is about 0.0065 and the validation loss is about 0.032.And The RTE and ATE results on the test set are close to those mentioned in the paper.
 Did you get similar result? Was my training overfitting?If you did the same thing, Please let me know your result and opinion.
